src/languages/es.ts

```diff
@@ -2237,6 +2238,21 @@ export default {
             control: 'Control',
             collect: 'Recolectar',
         },
+        expensifyCard: {
+            issueCard: 'Emitir tarjeta',
+            name: 'Nombre',
+            lastFour: '4 últimos',
+            limit: 'Limite',
+            currentBalance: 'Saldo actual',
+            currentBalanceDescription:
+                'El saldo actual es la suma de todas las transacciones contabilizadas con la Tarjeta Expensify que se han producido desde la última fecha de liquidación.',
+            remainingLimit: 'Límite restante',
+            requestLimitIncrease: 'Solicitud de aumento de límite',
```

Comment:
I need a confirmation for this translation

Reply:
@allgandalf Yeah, but there is no `Request limit increase` phrase in that list, so this one need to be confirmed

Reply:
Looks like we confirmed in thread: `Solicitar aumento de límite`